lib/jekyll-compose/file_creator.rb

```diff
@@ -0,0 +1,33 @@
+class Jekyll::Compose::FileCreator
```

Comment:
Why did you choose to do this instead of breaking out into multiple lines?

```ruby
module Jekyll
  module Compose
    class FileCreator
    end
  end
end
```